core/src/main/java/io/ap4k/processor/AptWriter.java

```diff
@@ -85,7 +81,9 @@ public void write(Configuration config) {
       name = name.toLowerCase();
       FileObject yml = processingEnv.getFiler().createResource(StandardLocation.CLASS_OUTPUT, PACKAGE, getProject().getAp4kOutputDir() + "/" + String.format(CONFIG, name, YML));
       try (Writer writer = yml.openWriter()) {
-        writer.write(Serialization.asYaml(config));
+        final String value = Serialization.asYaml(config);
+        writer.write(value);
+        return new AbstractMap.SimpleEntry<>(yml.toString(), value);
```

Comment:
The call to `toString()` kind of scares me.

I think that this is more acurate:
    
    Urls.toFile(yml.toUri().toURL()).getAbsolutePath();

Reply:
Also the javadoc should mention that we are talking about absolute path (we do talk about absolute path, don't we)?

Reply:
Yes, that is correct, it's the absolute path